js/collapse.js

```diff
@@ -16,7 +16,7 @@
   var Collapse = function (element, options) {
     this.$element      = $(element)
     this.options       = $.extend({}, Collapse.DEFAULTS, options)
-    this.$trigger      = $(this.options.trigger).filter('[href="#' + element.id + '"], [data-target="#' + element.id + '"]')
+    this.$trigger      = $('[data-toggle="collapse"]').filter('[href="#' + element.id + '"], [data-target="#' + element.id + '"]')
```

Comment:
Using a slightly more complex selector and getting rid of the filter intuitively seems like it would be faster, although I haven't microbenchmarked it.